vmm/src/config.rs

```diff
@@ -1635,6 +1635,14 @@ impl FsConfig {
             }
         }
 
+        if self.dax {
+            warn!(
+                "DAX feature is not supported as it is not stable yet. The \
+                provided value is ignored and DAX is disabled.\nThe option is \
+                deprecated from now on and will be removed in 2 releases."
```

Comment:
I would simplify this a bit. "The experimental DAX feature is deprecated and will be removed in a future release. A request to enable it is ignored."

Reply:
Done and done.